compiler/x/codegen/RealRegisterManager.hpp

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2017 IBM Corp. and others
```

Comment:
This looks like a new file. The initial copyright date seems incorrect. Similarly elsewhere in this PR.

Reply:
Thanks for pointing it out, it has been updated.